src/dagent/script/build_dagent.sh

```diff
@@ -6,7 +6,10 @@ cd ..
 rm -rf ./vendor/pnconnector
 cp -rf ../pnconnector ./vendor/
```

Comment:
Are these two lines required?

Reply:
yes, we want to copy fresh pnconnector directory inside vendor directory